This patch does not fix any bug. It solves some issues given by the build from sources
using the packaging standard in Slackbuilds.org that sets '/tmp/SBo/package-application'
as directory of installation ($PKG variable)
Do not apply if you install birdfont with a slackbuild modified using another path
for $PKG

--- ./install.py	2023-08-07 21:01:22.000000000 +0200
+++ ./install.py	2023-09-05 18:30:08.552843554 +0200
@@ -47,6 +47,6 @@
 	f = getDest (file, dir)
 	print ("install: " + f)
-	run ('install -d ' + dest + prefix + dir)
-	run ('install -m ' + str(mode) + ' ' + file + ' ' + dest + prefix + dir + '/')
+	run ('install -d ' + '/tmp/SBo/package-birdfont/' + dest + prefix + dir)
+	run ('install -m ' + str(mode) + ' ' + file + ' ' + '/tmp/SBo/package-birdfont/' + dest + prefix + dir + '/')
 	installed.write (f + "\n")
 
@@ -54,11 +54,11 @@
 	f = getDestRoot (file, dir)
 	print ("install: " + f)
-	run ('install -d ' + dest + dir)
-	run ('install -m ' + str(mode) + ' ' + file + ' ' + dest + dir + '/')
+	run ('install -d ' + '/tmp/SBo/package-birdfont/' + dest + dir)
+	run ('install -m ' + str(mode) + ' ' + file + ' ' + '/tmp/SBo/package-birdfont/' + dest + dir + '/')
 
 def link (dir, file, linkname):
 	f = getDest (linkname, dir)
 	print ("install link: " + f)
-	run ('cd ' + dest + prefix + dir + ' && ln -sf ' + file + ' ' + linkname)
+	run ('cd ' + '/tmp/SBo/package-birdfont/' + dest + prefix + dir + ' && ln -sf ' + file + ' ' + linkname)
 	installed.write (f + "\n")
 	
@@ -162,13 +162,13 @@
         install ('build/bin/libbirdfont.so.' + '${LIBbirdfont_VERSION}', '/lib', 644)
 elif os.path.isfile ('build/bin/libbirdfont.so.' + version.SO_VERSION):
-	install ('build/bin/libbirdfont.so.' + version.SO_VERSION, libdir, 644)
+	install ('build/bin/libbirdfont.so.' + version.SO_VERSION, libdir, 755)
 	link (libdir, 'libbirdfont.so.' + version.SO_VERSION, ' libbirdfont.so.' + version.SO_VERSION_MAJOR)
 	link (libdir, 'libbirdfont.so.' + version.SO_VERSION, ' libbirdfont.so')
 elif os.path.isfile ('build/libbirdfont.so.' + version.SO_VERSION):
-	install ('build/libbirdfont.so.' + version.SO_VERSION, libdir, 644)
+	install ('build/libbirdfont.so.' + version.SO_VERSION, libdir, 755)
 	link (libdir, 'libbirdfont.so.' + version.SO_VERSION, ' libbirdfont.so.' + version.SO_VERSION_MAJOR)
 	link (libdir, 'libbirdfont.so.' + version.SO_VERSION, ' libbirdfont.so')
 elif os.path.isfile ('build/bin/libbirdfont.' + version.SO_VERSION + '.dylib'):
-	install ('build/bin/libbirdfont.' + version.SO_VERSION + '.dylib', libdir, 644)
+	install ('build/bin/libbirdfont.' + version.SO_VERSION + '.dylib', libdir, 755)
 	link (libdir, 'libbirdfont.' + version.SO_VERSION + '.dylib', ' libbirdfont.dylib.' + version.SO_VERSION_MAJOR)
 	link (libdir, 'libbirdfont.' + version.SO_VERSION + '.dylib', ' libbirdfont.dylib')
@@ -180,13 +180,13 @@
         install ('build/bin/libbirdgems.so.' + '${LIBbirdgems_VERSION}', '/lib', 644)
 elif os.path.isfile ('build/bin/libbirdgems.so.' + version.LIBBIRDGEMS_SO_VERSION):
-        install ('build/bin/libbirdgems.so.' + version.LIBBIRDGEMS_SO_VERSION, libdir, 644)
+        install ('build/bin/libbirdgems.so.' + version.LIBBIRDGEMS_SO_VERSION, libdir, 755)
         link (libdir, 'libbirdgems.so.' + version.LIBBIRDGEMS_SO_VERSION, ' libbirdgems.so.' + version.LIBBIRDGEMS_SO_VERSION_MAJOR)
         link (libdir, 'libbirdgems.so.' + version.LIBBIRDGEMS_SO_VERSION, ' libbirdgems.so')
 elif os.path.isfile ('build/libbirdgems.so.' + version.LIBBIRDGEMS_SO_VERSION):
-        install ('build/libbirdgems.so.' + version.LIBBIRDGEMS_SO_VERSION, libdir, 644)
+        install ('build/libbirdgems.so.' + version.LIBBIRDGEMS_SO_VERSION, libdir, 755)
         link (libdir, 'libbirdgems.so.' + version.LIBBIRDGEMS_SO_VERSION, ' libbirdgems.so.' + version.LIBBIRDGEMS_SO_VERSION_MAJOR)
         link (libdir, 'libbirdgems.so.' + version.LIBBIRDGEMS_SO_VERSION, ' libbirdgems.so')
 elif os.path.isfile ('build/bin/libbirdgems.' + version.LIBBIRDGEMS_SO_VERSION + '.dylib'):
-        install ('build/bin/libbirdgems.' + version.LIBBIRDGEMS_SO_VERSION + '.dylib', libdir, 644)
+        install ('build/bin/libbirdgems.' + version.LIBBIRDGEMS_SO_VERSION + '.dylib', libdir, 755)
         link (libdir, 'libbirdgems.' + version.LIBBIRDGEMS_SO_VERSION + '.dylib', ' libbirdgems.dylib.' + version.LIBBIRDGEMS_SO_VERSION_MAJOR)
         link (libdir, 'libbirdgems.' + version.LIBBIRDGEMS_SO_VERSION + '.dylib', ' libbirdgems.dylib')
